test/parallel/test-http-client-abort-no-agent.js

```diff
@@ -0,0 +1,21 @@
+'use strict';
+const common = require('../common');
+const http = require('http');
+const net = require('net');
+
+const server = http.createServer(() => {
+  throw new Error('test invalidation');
```

Comment:
I originally used `common.fail()` but I honestly don't like to use it with a non string argument.
In this case it will be called with the `request` object and the error message will be something like `AssertionError: [object Object]`. I find it horrible.